recipes/minvar/2.1/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set version = "2.1" %}
-{% set build = 0 %}
+{% set build = 1 %}
```

Comment:
pinning on run-only dep can be removed (needs https://github.com/bioconda/bioconda-utils/pull/264 to make linter happy)